app/controllers/api/v1/visualization/clusters_controller.rb

```diff
@@ -0,0 +1,172 @@
+            type: :string
+          },
+          {
+            name: :consensus,
+            in: :query,
+            description: 'Statistical parameter to use for consensus, e.g. “mean”.  Omit parameter if not applying consensus parameter.',
+            type: :string,
+            enum: VALID_CONSENSUS_VALUES
+          }]
+
+        cluster_response_docs = [
+          {code: 200, description: 'Cluster visualization json object'},
+          {code: 401, description: ApiBaseController.unauthorized},
+          {code: 403, description: ApiBaseController.forbidden('view study')},
+          {code: 404, description: ApiBaseController.not_found(Study)},
+          {code: 406, description: ApiBaseController.not_acceptable},
+        ]
+
+        swagger_path '/studies/{accession}/clusters' do
+          operation :get do
+            key :tags, [
+                'Visualization'
+            ]
+            key :summary, 'Get the default cluster and its sub-clusters for a study'
+            key :description, 'Get the default cluster group and its constituent cluster annotations'
+            key :operationId, 'study_clusters_path'
+            cluster_param_docs.each { |doc| parameter(doc) }
+            cluster_response_docs.each { |doc| response doc[:code], doc }
+          end
+        end
+
+        # note that 'index' returns the default cluster, rather than a list of all clusters
+        # see the 'show' method for documentation of the return type
+        def index
+          begin
+            cluster = @study.default_cluster
+            if cluster.nil?
+              render json: {error: 'No default cluster exists'}, status: 422 and return
+            end
+            render json: self.class.get_cluster_viz_data(@study, cluster, params)
+          rescue => e
+            byebug
+          end
+        end
+
+        swagger_path '/studies/{accession}/clusters/{cluster_name}' do
+          operation :get do
+            key :tags, [
+                'Visualization'
+            ]
+            key :summary, 'Get a cluster and its sub-clusters for a study'
+            key :description, 'Get the cluster group and its constituent cluster annotations'
+            key :operationId, 'study_cluster_path'
+            parameter({
+              name: :cluster_name,
+              in: :path,
+              description: 'Name of cluster group',
+              required: true,
+              type: :string
+            })
+            cluster_param_docs.each { |doc| parameter(doc) }
+            cluster_response_docs.each { |doc| response doc[:code], doc }
+          end
+        end
+
+        def show
+          # this endpoint requires a cluster to be specified -- index is used to fetch the default
+          cluster = @study.cluster_groups.find_by(name: params[:cluster_name])
+          if cluster.nil?
+            render json: {error: "No cluster named #{params[:cluster_name]} could be found"}, status: 422 and return
```

Comment:
If the cluster cannot be found, the status should be `404` and not `422`.  `422` is for an "Unprocessable Entity" which is typically reserved for POST/PUT requests that contain validation errors.